pkg/quota/v1/evaluator/core/pods.go

```diff
@@ -354,6 +354,10 @@ func PodUsageFunc(obj runtime.Object, clock clock.Clock) (corev1.ResourceList, e
 		limits = quota.Max(limits, pod.Spec.InitContainers[i].Resources.Limits)
 	}
 
+	if feature.DefaultFeatureGate.Enabled(features.PodOverhead) {
```

Comment:
i _wish_ we had this in beta as intended happy we caught it prior to moving to GA.